test/total.jl

```diff
@@ -171,8 +171,8 @@ end
     
     # Test multiple domains passed at once
     tot = total(:api00, [:stype,:cname], dclus1_boot)
```

Comment:
What if you have 3 symbols?

Reply:
works as well, I just tried `tot = total(:api00, [:stype,:cname, :meals], dclus1_boot)`